src/generate/rust.rs

```diff
@@ -16,14 +16,14 @@ pub trait RustInputPat {
     fn rust_matcher(&self) -> Src;
 }
 
-impl<S: AsRef<str>> RustInputPat for ::scannerless::Pat<S> {
+impl<S: AsRef<str>> RustInputPat for crate::scannerless::Pat<S> {
```

Comment:
At this point, I'd say just import the `scannerless` module.

Reply:
Let's do that in a follow up?